tools/deprecate_features/deprecate_features.py

```diff
@@ -4,7 +4,6 @@
 import re
 import subprocess
 import fileinput
-from six.moves import input
```

Comment:
if we wan to keep `six` here we could depend on it from elsewhere (ie `base_pip3`)

i would prefer not to. Python >= 3.8 is an assumption baked into a lot of the tooling (even tho much will work regardless)

the pytooling repo tests against 3.8 and 3.9 currently